frontend/lib/store_widgets/detail/detail_content.dart

```diff
@@ -53,7 +53,6 @@ class DetailContent extends StatelessWidget {
               ],
               Column(
                 children: <Widget>[
-                  if (address != null)
```

Comment:
Indentation/linting? Is there an issue for this?

Reply:
https://www.elitenetzwerk.de/portal/user/home/main/query?eventId=2114

Reply:
What is this?

Reply:
ups wrong thing in clipboard :D

Reply:
I added a circleci job in this PR for validating that it is formatted